### PR TITLE
EN-2283 general settings tab

### PR DESCRIPTION
**Pull request by gidesan**
Adds the general settings tab to the page settings modal

https://jira.entando.org/browse/EN-2283

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/ui/pages/config/PageConfigPage.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import PageConfigGridContainer from 'ui/pages/config/PageConfigGridContainer';
 import ToolbarPageConfigContainer from 'ui/pages/config/ToolbarPageConfigContainer';
 import SelectedPageInfoTableContainer from 'ui/pages/common/SelectedPageInfoTableContainer';
 import { PAGE_STATUS_PUBLISHED } from 'state/pages/const';
-import PageSettingsModalContainer from 'ui/pages/config/PageSettingsModalContainer';
+import PageSettingsModalContainer from './SinglePageSettingsModalContainer';
```
Comment (nicpuddu) [inline src/ui/pages/config/PageConfigPage.js]: no relative imports

File: src/ui/pages/config/PageConfigPageContainer.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { getPageIsOnTheFly, getSelectedPageDiffersFromPublished, getSelectedPage
 import { getSelectedPage, getSelectedPageIsPublished, getSelectedPagePreviewURI } from 'state/pages/selectors';
 import { getLocale } from 'state/locale/selectors';
 import { setVisibleModal } from 'state/modal/actions';
-import { MODAL_ID } from 'ui/pages/config/PageSettingsModal';
+import { MODAL_ID } from './SinglePageSettingsModal';
```
Comment (nicpuddu) [inline src/ui/pages/config/PageConfigPageContainer.js]: no relative imports

File: src/ui/pages/config/SinglePageSettingsForm.js [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { Field, FieldArray, FormSection, reduxForm } from 'redux-form';
+import { Button, Col, Form, FormGroup, Nav, NavItem, Row, TabContainer, TabContent, TabPane } from 'patternfly-react';
+import { maxLength, required } from '@entando/utils';
+import RenderTextInput from 'ui/common/form/RenderTextInput';
+import FormLabel from 'ui/common/form/FormLabel';
+import SwitchRenderer from 'ui/common/form/SwitchRenderer';
+import RenderSelectInput from 'ui/common/form/RenderSelectInput';
+import MultiSelectRenderer from '../common/MultiSelectRenderer';
```
Comment (nicpuddu) [inline src/ui/pages/config/SinglePageSettingsForm.js]: no relative imports

**Comment by entando-service** [comment]
All is well

File: src/ui/pages/config/PageConfigPage.js (hunk 1 above)
Comment (gidesan) [reply src/ui/pages/config/PageConfigPage.js]: fixed

File: src/ui/pages/config/PageConfigPageContainer.js (hunk 2 above)
Comment (gidesan) [reply src/ui/pages/config/PageConfigPageContainer.js]: fixed

File: src/ui/pages/config/SinglePageSettingsForm.js (hunk 3 above)
Comment (gidesan) [reply src/ui/pages/config/SinglePageSettingsForm.js]: fixed

**Pull request merged**